recipes/pythesint/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pythesint" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "457ae24b8ffcee7a03ff9515c1578aec81d2bf3dd7a599d7cafc6e146e441b65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not py27 or win]
+
+requirements:
+  build:
+    - python
+    - pyyaml
+    - requests
+    - setuptools
+  run:
+    - python
+    - pyyaml
+    - requests
+
+test:
+  requires:
+    - mock
+    - nose
+  commands:
+    - nosetests pythesint
+
+about:
+  home: https://github.com/nansencenter/py-thesaurus-interface
+  license: GPL-3.0
+  license_family: GPL
+  # TODO: add license file to the repo
+  summary: 'An interface to metadata vocabularies for geospatial and other geophysical data'
+
+  description: |
+    pythesint provides access to controled vocabularies following
+    the standards defined at NASA's Global Change Master Directory (GCMD)
+    (http://gcmd.gsfc.nasa.gov) and the NetCDF-CF conventions
+    (http://cfconventions.org/), plus possibly others that will be added as
+    needs emerge. The standards are mapped in Python dictionaries and saved
+    to json-files.
+  doc_url: https://github.com/nansencenter/py-thesaurus-interface
+  dev_url: https://github.com/nansencenter/py-thesaurus-interface
+
+extra:
+  recipe-maintainers:
+    - akorosov
+    - aleksandervines
```

Comment:
@aleksandervines -- are you ok being added as a co-maintainer of the `pythesint` recipe on conda-forge?

Reply:
I was the one commiting this, so yes :)